app/views/about/more.json.rabl

```diff
@@ -0,0 +1,10 @@
+node(:uri)                  { site_hostname }
+node(:title)                { Setting.site_title }
```

Comment:
I was actually thinking the same, with regards to (c). Since the api is all about returning machine-readable data, it would make sense to keep all the machine-readable bits in one place there.

I'll submit a PR for that quickly, and reference this one.